src/content/effects.js

```diff
@@ -0,0 +1,129 @@
+import { items } from './items';
+import { playerStatus } from './playerStatus';
+import { specialItems } from './specialItems';
+import { skills } from './skills';
+import { getRandomKey } from '../utility/getRandomKey';
+import { getDomElement } from '../utility/getDomElement';
+import { killChildren } from '../utility/killChildren';
+import { renderScene } from '../utility/renderScene';
+import { changeStatusIndicators } from '../utility/changeStatusIndicators';
+import { killPlayer } from '../utility/killPlayer';
+import { getRandomDirection } from '../utility/getRandomDirection';
+import {
+  inventoryHolder, menu, gameplayUI, skillsHolder,
+} from '../utility/domElements';
+
+
+import { removeSkills } from '../utility/removeSkills';
+
+const effects = {
+  getRandomItem: (quantity) => {
+    for (let i = 1; i <= quantity; i += 1) {
```

Comment:
Don't combine business logic and dom manipulations. Follow the single responsibility principle.